src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -839,6 +852,21 @@ cdef class Font:
             draw_state_p = <void *>PyCapsule_GetPointer(draw_state, NULL)
         hb_font_draw_glyph(self._hb_font, gid, draw_funcs._hb_drawfuncs, draw_state_p);
 
+    def paint_glyph(self, gid: int,
+                    paint_funcs: PaintFuncs,
+                    paint_state: object = None,
+                    palette_index: int = 0,
+                    foreground: int = 0x000000FF) -> None:
```

Comment:
Should we have a nicer representation of colors than raw integers?

Reply:
Probably a named tuple?

Reply:
I don't have a preference honestly.